app/javascript/components/episodes/EditEpisode.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import axios from 'axios';
+import { Form, Input, DatePicker, Button } from 'antd';
+import moment from 'moment';
+
+class EditEpisode extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { id: '', title: '', date: '', video: '', description: '' };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  };
+  
+  componentDidMount () {
+    const { id } = this.props.match.params
+
+    axios.get(`/api/v1/episodes/${id}`)
+      .then(response => {
+        this.setState({ 
+          id: response.data.id,
+          title: response.data.title,
+          date: response.data.date,
+          video: `https://www.youtube.com/watch?v=${response.data.video}`,
+          description: response.data.description
+         })
+      })
+      .catch(function (error) {
+        console.log(error);
+    });
+  };
+  
+  handleChange(event) {
+    this.setState({[event.target.name]: event.target.value}, () => console.log(this.state));
+  };
+
+  handleSubmit = e => {
+    e.preventDefault();
+    const { id, title, date, video, description } = this.state
+    
+    const options = {
+      method: 'patch',
+      url: `/api/v1/episodes/${id}`,
+      data: {
+        title: title,
+        date: date,
+        video: video,
+        description: description
+      }
+    };
+    axios(options)
+      .then(response => {
+        console.log(response);
+        this.props.history.push('/admin/episodes');
+    })
+  };
+
+  render() {
+    const { title, date, video, description } = this.state
+    return (
+      <div class="container">
```

Comment:
Should be a component like:
```js
const BootstrapComponents = { // or using class
  Container: function(...) {
    return React.createElement(...)
  },
  ...
}
```
As well for other bootstrap elements such as `row`.
Then you should build your page using that components without plain html
For more information check this: https://reactjs.org/docs/jsx-in-depth.html#using-dot-notation-for-jsx-type